core/vm/logger.go

```diff
@@ -178,10 +183,12 @@ func (l *StructLogger) CaptureState(env *EVM, pc uint64, op OpCode, gas, cost ui
 	return nil
 }
 
+// CaptureFault logs a fault in the execution
```

Comment:
// CaptureFault implements the Tracer interface to trace an execution fault
// while running an opcode.